plugins/sm-operator/sm-operator/templates/operator.yaml

```diff
@@ -0,0 +1,338 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: sagemaker-k8s-operator-leader-election-role
+  namespace: {{ .Release.Namespace | default "sagemaker-k8s-operator-system" | quote }}
```

Comment:
top level items must have the plugins shared labels , or future installs will have issues with upgrading. (missing helm labels)

Reply:
Show me an example and I can implement